src/client/Microsoft.Identity.Client/OAuth2/Throttling/ThrottleCommon.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Internal.Requests;
+
+namespace Microsoft.Identity.Client.OAuth2.Throttling
+{
+    internal static class ThrottleCommon
+    {
+        public const string ThrottleRetryAfterHeaderName = "x-ms-lib-capability";
+        public const string ThrottleRetryAfterHeaderValue = "retry-after, h429";
+
+        internal const string KeyDelimiter = ".";
+
+        /// <summary>
+        /// The strict thumbprint is based on: 
+        /// ClientId
+        /// Authority
+        /// Resource
```

Comment:
maybe add a comment that resource is not applicable for msal? for future team members?